src/modules/displayShows.js

```diff
@@ -0,0 +1,5 @@
+export default async function fetchData(id) {
+  const fetching = await fetch(`https://api.tvmaze.com/shows/${id}`)
+    .then((response) => response.json());
+  return fetching;
+}
```

Comment:
- [ ] Please, It's one of the project requirements you use `arrow` syntax for functions instead of JavaScript Function Expressions because in this project you learn how to use ES6 new features. Kindly make this :wink: 💯 

- For more documentation, you can visit this [link](https://www.w3docs.com/learn-javascript/function-expressions.html)

Reply:
done